### ISSUE TITLE
Optimization

### ISSUE THREAD

**Issue by DavidWigley** [DavidWigley/DavidWigley.github.io#2]
The file latheTest.js is very resource intensive when you need precise shapes (ie a sphere with no edges) due to the math needed to make this happen.

Need to find a way to optimize it. One way I'm thinking is to make our trace a full circle rather than a half circle and then only lathe a half revolution (pi rads) rather than a full rev.

**Comment by DavidWigley** [comment]
It looks like that is how the devs made the sphere geometry although theirs has a height and width component. For mine the for loop for generating points can be seen as height. May make dynamic in the future.

Anyways, I did implement what I stated earlier.

https://github.com/DavidWigley/DavidWigley.github.io/commit/c28f9373c8008582f2478816571155a4fc850f29
Evident in that commit. No drastic performance change unfortunately.
